src/services/decoders/ClpIrDecoder/index.ts

```
@@ -1,46 +1,55 @@
-import clpFfiJsModuleInit, {ClpStreamReader} from "clp-ffi-js";
+import clpFfiJsModuleInit, {ClpStreamReader, MainModule} from "clp-ffi-js";
```

ESLint

[error] Imports must be broken into multiple lines if there are more than 1 elements.

(import-newlines/enforce)